draft-ietf-dnsop-structured-dns-error-page.xml

```diff
@@ -0,0 +1,1003 @@
+<!-- control references -->
+<?rfc symrefs="yes"?>
+<!-- use symbolic references tags, i.e, [RFC2119] instead of [1] -->
+<?rfc sortrefs="yes" ?>
+<!-- sort the reference entries alphabetically -->
+<!-- control vertical white space 
+     (using these PIs as follows is recommended by the RFC Editor) -->
+<?rfc compact="yes" ?>
+<!-- do not start each main section on a new page -->
+<?rfc subcompact="no" ?>
+<!-- keep one blank line between list items -->
+<!-- end of list of popular I-D processing instructions -->
+<rfc category="std" consensus="true"
+     docName="draft-ietf-dnsop-structured-dns-error-page-00" ipr="trust200902"
+     obsoletes="" sortRefs="true" submissionType="IETF" symRefs="true"
+     tocDepth="4" tocInclude="true" updates="8914" version="3" xml:lang="en">
+  <!-- xml2rfc v2v3 conversion 3.9.1 -->
+
+  <front>
+    <title abbrev="Data for Filtered DNS">Structured Data for Filtered
+    DNS</title>
+
+    <seriesInfo name="Internet-Draft" value="" />
+
+    <author fullname="Dan Wing" initials="D." surname="Wing">
+      <organization abbrev="Citrix">Citrix Systems, Inc.</organization>
+
+      <address>
+        <postal>
+          <street />
+
+          <country>United States of America</country>
+        </postal>
+
+        <email>dwing-ietf@fuggles.com</email>
+      </address>
+    </author>
+
+    <author fullname="Tirumaleswar Reddy" initials="T." surname="Reddy">
+      <organization abbrev="Nokia">Nokia</organization>
+
+      <address>
+        <postal>
+          <street />
+
+          <city>Bangalore</city>
+
+          <region>Karnataka</region>
+
+          <code />
+
+          <country>India</country>
+        </postal>
+
+        <email>kondtir@gmail.com</email>
+      </address>
+    </author>
+
+    <author fullname="Neil Cook" initials="N." surname="Cook">
+      <organization>Open-Xchange</organization>
+
+      <address>
+        <postal>
+          <street />
+
+          <country>United Kingdom</country>
+        </postal>
+
+        <email>neil.cook@noware.co.uk</email>
+      </address>
+    </author>
+
+    <author fullname="Mohamed Boucadair" initials="M." surname="Boucadair">
+      <organization>Orange</organization>
+
+      <address>
+        <postal>
+          <street />
+
+          <city>Rennes</city>
+
+          <region />
+
+          <code>35000</code>
+
+          <country>France</country>
+        </postal>
+
+        <email>mohamed.boucadair@orange.com</email>
+      </address>
+    </author>
+
+    <date />
+
+    <workgroup>DNSOP WG</workgroup>
+
+    <abstract>
+      <t>DNS filtering is widely deployed for network security, but filtered
+      DNS responses lack information for the end user to understand the reason
+      for the filtering. Existing mechanisms to provide detail to end users
+      cause harm especially if the blocked DNS response is to an HTTPS
+      website.</t>
+
+      <t>This document updates the EXTRA-TEXT field of Extended DNS Error to
```

Comment:
I suggest to merge these two sentence to highlight what we actually update in 8914:

This document updates RFC 8914 by structuring the EXTRA-TEXT field of the Extended DNS Error to provide details on the DNS filtering. Such details can be parsed by the client and displayed, logged, or used for other purposes. Other than that, this document does not change any thing written in RFC 8914.

"This document updates RFC 8914 by structuring the EXTRA-TEXT field of the Extended DNS Error to provide details on the DNS filtering. Such details can be parsed by the client and displayed, logged, or used for other purposes. Other than that, this document does not change any thing written in RFC 8914."